examples/example_direct_damage_manual.ipynb

```diff
@@ -0,0 +1,264 @@
+    "- Huizinga\n",
+    "- OSdaMage \n",
+    "- User defined"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## An example for a manual damage function"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this notebook, we provide an example for an event-based object-oriented Huizinga analysis."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import geopandas as gpd\n",
+    "import matplotlib.pyplot as plt\n",
+    "from pathlib import Path\n",
+    "import rasterio\n",
+    "from ra2ce.ra2ce_handler import Ra2ceHandler #import the ra2cehandler to run ra2ce analyses\n",
+    "\n",
+    "root_dir = Path(\"data\", \"direct_analysis_manual\")\n",
+    "assert root_dir.exists(), \"root_dir not found.\""
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Introducing a hazardous event"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To use the flood map with RA2CE, we need to fill in the **[hazard]** section in the network.ini. \n",
+    "\n",
+    "Specify the flood map name in the **hazard_map** parameter in network.ini. RA2CE expects the flood map to be located in the *hazard* folder. The **aggregate_wl** parameter in analysis.ini can be set to either 'max', 'min' or 'mean' to take the maximum, minimum or mean water depth per road segment when the exposure of the roads to a certain hazard (map) is determined.<br>\n",
+    "\n",
+    "Set the right CRS for the flood map in the **hazard_crs** parameter. This CRS can be different from the origins, destinations and road network. RA2CE will reproject the network to the CRS of the flood map and will reproject the road back to the original CRS when the CRS differs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hazard_folder = root_dir / \"static\" / \"hazard\" # find the hazard folder where you locate your floo dmap\n",
+    "hazard_map = hazard_folder / \"max_flood_depth.tif\" # set the location of the hazard map\n",
+    "\n",
+    "# Open the TIF file using rasterio\n",
+    "with rasterio.open(hazard_map) as src:\n",
+    "    # Read the TIF file as a numpy array\n",
+    "    tif_array = src.read(1)  # Change the band index (1) if necessary\n",
+    "\n",
+    "plt.figure(figsize=(10, 10))\n",
+    "plt.imshow(tif_array, cmap='Blues')  # Change the colormap if desired\n",
+    "plt.colorbar(label='Pixel Values')\n",
+    "plt.title('Flood map')\n",
+    "plt.show()\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Specifying the .ini files"
```

Comment:
Keep in mind with the current approach of ra2ce you no longer need to use `ini` files in order to generate network or damages analysis.